tutorials/quick_start/README.md

```diff
@@ -37,24 +37,31 @@ In this example, we are running the main.py script with the following parameters
 - `--validation_dataset_folder`: Specifies the path to the local copy of the dataset to be used for evaluation.
 - `--representative_dataset_folder`: Specifies the path to the local copy of the dataset to be used for quantization.
 
-#### Set of models quantization example
-In the following example, we evaluate a set of pre-trained models. Assuming that the necessary packages have been installed (as mentioned earlier), we use the following command:
-```python
-python main.py --models_list_csv <myfile.csv>
-```
-The content of the CSV file consists of a tabular structure where each line represents a model along with its corresponding parameters to be used. For instance:
-
-| model_name    | model_library | dataset_name  | validation_dataset_folder     | representative_dataset_folder     |
-|---------------|---------------|---------------|-------------------------------|-----------------------------------|
-| mobilenet_v2  | torchvision   | ImageNet      | /path/to/validation/dataset   | /path/to/representative/dataset   |
-| regnetx_002   | timm          | ImageNet      | /path/to/validation/dataset   | /path/to/representative/dataset   |
+Please note that the above example assumes the command is run from the command line and that the [main.py](./main.py) script is in the current directory.
 
+Make sure to refer to the script itself to understand all available parameters and their usage.
 
+#### Advanced model quantization example
+##### Mixed-precision 
+In this example, we employ the MCT Mixed-Precision quantization workflow to achieve a more aggressive reduction in the model's size. 
```

Comment:
Also, not sure about the word "aggressive".
Maybe say something like "to further reduce the model's size, with minimal reduction in the quantized model accuracy"